### DISCUSSION TITLE
HLS VOD support?

### DISCUSSION THREAD

**Discussion by KudryavtsevDmitry** [bluenviron/mediamtx#875] [Questions]
Hello. I have several questions about features

Does it support HLS VOD? If so, does it make it on-the-fly packaging (without writing/storing chunks on disk)
Does it support mp4, mp3 containers?

**Comment by aler9** [comment]
Hello,

> Does it support HLS VOD? If so, does it make it on-the-fly packaging (without writing/storing chunks on disk)

Yes, HLS is generated on-the-fly and nothing is stored on disk, everything is kept on RAM in order to maximize performance.

> Does it support mp4, mp3 containers?

You can ingest any kind of file you want into the server, just use ffmpeg to convert it into the RTSP or RTMP streaming format.

[a comment by an automated account is not shown]
